lib/pry/color.rb

```diff
@@ -0,0 +1,97 @@
+module Pry::Color
+  extend self
+  COLORS = {
+    "black"   => 0,
+    "red"     => 1,
+    "green"   => 2,
+    "yellow"  => 3,
+    "blue"    => 4,
+    "purple"  => 5,
+    "magenta" => 5,
+    "cyan"    => 6,
+    "white"   => 7
+  }
+  BRUSH_METHODS = [*COLORS.keys, *["bold", "default", "bright_default", "strip_color"]]
```

Comment:
what’s the point in splatting out an array? why not just inline the: `”bold”`, etc ?

Reply:
because we don't want an array of arrays here, we want all brush methods contained within a single array.

Reply:
This looks simpler: `COLORS.keys + %w(bold default bright_default strip_color)`